dxram/src/main/java/com/yahoo/ycsb/db/YCSBObject.java

```diff
@@ -0,0 +1,103 @@
+package com.yahoo.ycsb.db;
+
+import com.yahoo.ycsb.ByteIterator;
+
+import de.hhu.bsinfo.dxmem.data.AbstractChunk;
+import de.hhu.bsinfo.dxutils.serialization.Exporter;
+import de.hhu.bsinfo.dxutils.serialization.Importer;
+
+/**
+ * Data structure for YCSB objects.
+ *
+ * @author Stefan Nothaas, stefan.nothaas@hhu.de, 09.11.2018
+ * @author Kevin Beineke, kevin.beineke@hhu.de, 22.04.2017
```

Comment:
denote authorship through git commit message ([use the github feature for multiple authors if needed](https://help.github.com/en/articles/creating-a-commit-with-multiple-authors))